cypress/fixtures/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "next-on-netlify-test",
   "scripts": {
-    "build": "../../../node_modules/.bin/next build",
```

Comment:
This relies on shebangs, which do not work on Windows.
This PR instead relies on `execa`, which adds shebangs support for Windows.